test/tools/libtesteth/ImportTest.cpp

```diff
@@ -55,10 +56,13 @@ bytes ImportTest::executeTest()
 	if (m_testType == testType::StateTests)
 	{
 		eth::Network network = eth::Network::MainNetwork;
-		std::pair<eth::State, ImportTest::execOutput> out = executeTransaction(network, m_envInfo, m_statePre, m_transaction);
-		m_statePost = out.first;
-		m_logs = out.second.second.log();
-		return out.second.first.output;
+		LogEntries emptyLogs;
+		ExecutionResult emptyRes;
+		ImportTest::ExecOutput execOut = make_pair(emptyRes, TransactionReceipt(h256(), u256(), emptyLogs));
+		std::tie(m_statePost, execOut, std::ignore) =
```

Comment:
Here this change was not necessary as ExecutionResult does not have default constructor, but it can stay.